master/docs/manual/cfg-www.rst

```diff
@@ -476,17 +476,71 @@ Here is a configuration that is known to work (Apache 2.4.10 / Debian 8), direct
 Authorization rules
 ~~~~~~~~~~~~~~~~~~~
 
+The authorization framework in Buildbot is very generic and flexible.
+Drawback is that it is not very obvious for new comers.
+The 'simple' example will however allow you to easily start by implementing an admins-have-all-rights setup.
+
+Please carefully read the following documentation to understand how to setup authorization in Buildbot.
+
+Authorization framework is tightly coupled to the REST API.
+Authorization framework only works for HTTP, not for other means of interaction like IRC or try scheduler.
+It allows or denies access to the REST APIs according to rules.
+
+.. blockdiag::
+
+    blockdiag {
+      User -> AuthenticatedUser [label = Auth];
+      AuthenticatedUser -> "RoleMatcher" -> Role <- "EndpointMatcher" <- "REST API Endpoint"
+
+      User  [shape = actor];
+      AuthenticatedUser  [shape = actor];
+      RoleMatcher [shape = diamond];
+      EndpointMatcher [shape = diamond];
+    }
+
+- Roles is a label that you give to a user.
+  It is similar to the notion of group, but with more flexibility.
+  For an example of how it is different to group, there is the ``owner`` role, which can be given to a user for a build that he is at the origin.
+
+- Endpoint matchers associates role requirements to REST API endpoints.
```

Comment:
"associate"

Maybe clarify the default policy when nothing matches? (reject by default, but there seems to be a special case when the list is empty (allow all). I see it is mentioned below, but for a global picture I think it is an important detail.